Experimental/DistributedAuthoritySample/Assets/Scripts/Input/AvatarInputs.cs

```diff
@@ -0,0 +1,76 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace com.unity.multiplayer.samples.distributed_authority.input
+{
+    public class AvatarInputs : MonoBehaviour
+    {
+        [Header("Character Input Values")]
+        public Vector2 move;
+        public Vector2 look;
+        public bool jump;
+        public bool sprint;
+
+        [Header("Movement Settings")]
+        public bool analogMovement;
+
+        [Header("Mouse Cursor Settings")]
+        public bool cursorLocked = true;
+        public bool cursorInputForLook = true;
```

Comment:
1. Do these need to be public, or would it be good if they were `[SerializeField]` + internal?
2. if they must be public, they don't seem to follow unity's standard coding conventions, and they need to be documented (///)

Reply:
Great! Good! Then we should also adjust the naming to be PascalCase according to the [coding conventions](https://github.cds.internal.unity3d.com/unity/iet-package/blob/dev/Docs/CSharpReference.cs) (`internal` is "public within namespace").
I.E: cursorLocked  -> CursorLocked

Reply:
Huh, it's not enforced by our .editorconfig, and getting Rider & our dotnet solution to reformat doesn't apply the change.
Regardless, I've modified it!